### PR TITLE
Bugfix. Switch off quickly all AQO features if queryId is disabled.

### PR DESCRIPTION
**Pull request by danolivo**
One installcheck test was added into the github actions workflow.

Push it into stable14 and master as quick as possible.
stable13 based on custom query hash and don't needed this.
Remember! The master branch can contain one more hook! Check it.

### CODE REVIEW

**Review by Anisimov-ds — COMMENTED** [review]
Вопросы только по объявленным, но неиспользуемым хукам в cardinality_hooks.c

File: aqo.c [hunk 1]
```diff
@@ -98,19 +98,20 @@ MemoryContext 		AQOLearnMemCtx = NULL;
...
-set_baserel_rows_estimate_hook_type			prev_set_baserel_rows_estimate_hook;
-get_parameterized_baserel_size_hook_type	prev_get_parameterized_baserel_size_hook;
-set_joinrel_size_estimates_hook_type		prev_set_joinrel_size_estimates_hook;
-get_parameterized_joinrel_size_hook_type	prev_get_parameterized_joinrel_size_hook;
-ExplainOnePlan_hook_type					prev_ExplainOnePlan_hook;
-ExplainOneNode_hook_type					prev_ExplainOneNode_hook;
+/* Saved hooks */
+static planner_hook_type						prev_planner_hook							= NULL;
+static ExecutorStart_hook_type					prev_ExecutorStart_hook						= NULL;
+static ExecutorRun_hook_type					prev_ExecutorRun							= NULL;
```
[the remark this answers is not in the document]
Comment (danolivo) [reply aqo.c]: Done

File: cardinality_hooks.c [hunk 2]
```diff
@@ -33,103 +33,16 @@
 #include "machine_learning.h"
 #include "path_utils.h"
 
-estimate_num_groups_hook_type prev_estimate_num_groups_hook = NULL;
+estimate_num_groups_hook_type				prev_estimate_num_groups_hook				= NULL;
+set_baserel_rows_estimate_hook_type			prev_set_baserel_rows_estimate_hook			= NULL;
+get_parameterized_baserel_size_hook_type	prev_get_parameterized_baserel_size_hook	= NULL;
+set_joinrel_size_estimates_hook_type		prev_set_joinrel_size_estimates_hook		= NULL;
+get_parameterized_joinrel_size_hook_type	prev_get_parameterized_joinrel_size_hook	= NULL;
```
[the remark this answers is not in the document]
Comment (danolivo) [reply cardinality_hooks.c]: Done

File: cardinality_hooks.c [hunk 3]
```diff
@@ -33,103 +33,16 @@
 #include "machine_learning.h"
 #include "path_utils.h"
 
-estimate_num_groups_hook_type prev_estimate_num_groups_hook = NULL;
+estimate_num_groups_hook_type				prev_estimate_num_groups_hook				= NULL;
```
[the remark this answers is not in the document]
Comment (danolivo) [reply cardinality_hooks.c]: Не согласен. Кто-то может позже переопределить хук. Добавил более строгую проверку

**Review by Anisimov-ds — APPROVED** [review]

**Pull request merged**